### DISCUSSION TITLE
Table2Geo Country Names (ISO2, ISO3, FIPS)

### DISCUSSION THREAD

**Discussion by arky** [neocarto/bertin#40] [Q&A] [answered]
Wonder if it is possible to map country name ( Country name code:  CNT2/CNT3, ISO2/ISO3 or FIPS region IDS) using Table2Geo. 

Below is a good use case of quickly adding OWID data to map. 
Data: https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/vaccinations/locations.csv
                

![image](https://user-images.githubusercontent.com/808895/160518565-ab35cd55-7abf-44f8-a5e2-655572c5aada.png)

Visualization: https://data.opendevelopmentmekong.net/odm-short-url/1400ea3f-8143-45e2-add5-0ef5123ce4ae

**Comment by neocarto** [comment, marked as answer]
For that, I started working on another package. See https://github.com/neocarto/geocountries & https://observablehq.com/@neocartocnrs/geocountries

**Reply by arky** [reply]
@neocarto I haven't looked at geocountries. How do you use it with bertin, is there a tutorial somewhere?

**Reply by neocarto** [reply]
No I don't use it in bertinjs. It's a very small package to improve. There is a tutu here: https://observablehq.com/@neocartocnrs/geocountries
